src/averageAges.js

```diff
@@ -13,15 +13,30 @@
  * @param {number} century - optional
  *
  * @return {number}
+ *
  */
+function calculateAverage(array) {
+  return array.reduce((a, b) => a + b, 0) / array.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let mens = people.filter(person => person.sex === 'm');
```

Comment:
Для чого робити два рази сорт. Зайві ітерації))

```suggestion
  let mens = people.filter(person => century
  	? Math.ceil(person.died / 100) === century && person.sex === 'm'
  	: person.sex === 'm');
```